plugins/module_utils/network/ios/argspec/bgp_global/bgp_global.py

```diff
@@ -349,6 +349,7 @@ class Bgp_globalArgs(object):  # pylint: disable=R0903
                 "neighbor": {
                     "type": "list",
                     "elements": "dict",
+                    "aliases": ["neighbors"],
```

Comment:
We should be rendering as `neighbors` too in facts. It will probably not be a breaking change since both the words are accepted by the module.